client/app/contexts/cozy/index.json

```diff
@@ -17,5 +17,49 @@
     "figure": "facture-telephonique-lg.jpg",
     "color": {"css": "#1FA8F1"},
     "important": true
+  },
+  {
+    "slug": "fai-bills",
+    "connectors": [{
+      "slug": "bouyguesbox"
+    }, {
+      "slug": "free"
+    }, {
+      "slug": "numericable"
+    }, {
+      "slug": "sfr_box"
+    }],
+    "figure": "facture-fournisseur-internet-lg.jpg",
+    "color": {"css": "#1FA8F1"},
+    "important": true
+  },
+  {
+    "slug": "contacts",
+    "connectors": [{
+      "slug": "googlecontacts"
+    }, {
+      "slug": "linkedin"
+    }],
+    "figure": "regroupement-contacts-lg.jpg",
+    "color": {"css": "#1FA8F1"},
+    "important": true
+  },
+  {
+    "slug": "agenda",
+    "connectors": [ {
+      "slug": "doctolib"
+    },{
+      "slug": "facebook_events"
+    }, {
+      "slug": "ical_feed"
+    },{
+      "slug": "meetup"
+    }, {
+      "slug": "voyages_sncf"
+    }],
+    "figure": "regroupement-agendas-lg.jpg",
+    "color": {"css": "#1FA8F1"},
+    "important": true
   }]
+
```

Comment:
Maybe we could add an "other-bills" category with : aprr darty digitalocean directenergie edf electrabel github kimsufi_ca kimsufi_eu malakoff_mederic materiel_net numericable online_net ovh_ca ovh_eu runabove sosh soyoustart_ca soyoustart_eu trainline uber vente_privee voyages_sncf